app/index.js

```diff
@@ -1018,10 +1018,12 @@ module.exports = JhipsterGenerator.extend({
             // normal CSS or SCSS?
             if (this.useSass) {
                 this.template(WEBAPP_DIR + 'scss/main.scss', WEBAPP_DIR + 'scss/main.scss');
+                this.template(WEBAPP_DIR + 'scss/bootstrap.scss', WEBAPP_DIR + 'scss/bootstrap.scss');
             }
```

Comment:
I think that would be confusing? This file only contains bootstrap stuff, and there might in the future be more than this single vendor-supplied (bower) component?